clojure_liberator/src/clojure_liberator/core.clj

```diff
@@ -49,19 +52,28 @@
 (defresource create-client
```

Comment:
Best idea I could come up with was extracting the two anonymous functions to named ones, but :shrug: